pages/linux/inav.md

```diff
@@ -0,0 +1,16 @@
+# lnav
+
+> Advanced log file viewer for the console.
+> More information: <https://docs.lnav.org/en/latest/>.
+
+- Load and follow the system syslog file:
+
+`lnav`
+
+- Load all of the files in `/var/log`:
+
+`lnav /var/log`
+
+- Load all of the files in `/var/log`:
+
+`make 2>&1 | lnav -t`
```

Comment:
Is there a Makefile to execute `make 2>&1`?

Reply:
I don't know, I pasted the example given in this [issue](https://github.com/tldr-pages/tldr/issues/9995)